pull-request.sh

```diff
@@ -17,6 +17,7 @@ HEADER="${HEADER}; application/vnd.github.antiope-preview+json; application/vnd.
 # URLs
 REPO_URL="${BASE}/repos/${GITHUB_REPOSITORY}"
 PULLS_URL=$REPO_URL/pulls
+ISSUE_URL=$REPO_URL/issues
 
```

Comment:
I'm aware that pulls are considered a subtype of issue, but why did you specifically add this / change this here? Did you test without it and found that it didn't work?

Reply:
PR operations after creation goes to issues endpoint as per the docs :) I haven't tried calling the same url tho :)